x-pack/plugins/observability_solution/apm/server/routes/span_links/get_linked_parents.ts

```diff
@@ -38,6 +37,7 @@ export async function getLinkedParentsOfSpan({
     },
     _source: [SPAN_LINKS],
     body: {
+      fields: [SPAN_LINKS],
```

Comment:
we should not use fields for span links, it will break for APM events as well, we can just use source